config/config.go

```diff
@@ -404,6 +407,7 @@ type Experimental struct {
 	AuditLog              AuditLog              `yaml:"auditLog"`
 	AwsEnvironment        AwsEnvironment        `yaml:"awsEnvironment"`
 	AwsNodeLabels         AwsNodeLabels         `yaml:"awsNodeLabels"`
+	ClusterAutoScaler     ClusterAutoScaler     `yaml:"clusterAutoScaler"`
```

Comment:
Could you also make these `clusterAutoscalerSupport` in cluster.yaml and `ClusterAutoscalerSupport` in golang code respectively so that we can be consistent among other experimental features to support k8s addons like #297